snorkel/classification/snorkel_classifier.py

```diff
@@ -94,9 +94,16 @@ def __init__(
         # Build network with given tasks
         self._build_network(tasks)
 
+        # Report total task count and duplicates
+        all_ops = [op.name for t in tasks for op in t.task_flow]
+        unique_ops = set(all_ops)
+        all_mods = [mod_name for t in tasks for mod_name in t.module_pool]
+        unique_mods = set(all_mods)
         logging.info(
             f"Created multi-task model {self.name} that contains "
-            f"task(s) {self.task_names}."
+            f"task(s) {self.task_names} from "
+            f"{len(unique_ops)} operations ({len(all_ops) - len(unique_ops)} shared) and "
```

Comment:
I am not very familiar with pytorch, so ignore if this seems incorrect, but I think
f"{len(all_ops)} total ops, {len(unique_ops) unique ops} and"
f"{len(all_mods)} total modules, {len(unique_mods) unique modules}"
is more understandable.

Reply:
It's a good question, but in this case, I think it makes more sense to talk about shared modules. e.g., if I have a shared body and five unshared heads, the more important piece of info to me is that of my 6 modules, only 1 of them is shared between more than one task.